README.md

```diff
@@ -10,6 +10,7 @@ The snapshots are available in 2 formats: as MHTML files and as [WebTraversalLib
 The MHTML format is less lossy, a browser can render these pages though any Javascript on the page is lost.
 The WTL snapshots are produced by loading the MHTML pages into a chromium-based browser. To keep the WTL dataset more compact, they do not contain the screenshot of the rendered MTHML, but only its HTML, and page and element metadata with additional rendering information (bounding boxes of elements, font sizes etc.).
```

Comment:
When we say "they do not contain the screenshot", you can mention that "they are provided separately (see below)" (or something to that effect).